addons/calendar_sms/models/calendar_alarm.py

```diff
@@ -15,6 +15,8 @@ class CalendarAlarm(models.Model):
         domain=[('model', 'in', ['calendar.event'])],
         compute='_compute_sms_template_id', readonly=False, store=True,
         help="Template used to render SMS reminder content.")
+    force_remind_responsible = fields.Boolean(
```

Comment:
Why "force" ? Can it just be "remind_responsible" ? Or maybe "notify_responsible" ?
"remind" is a bit strange, would go for "Notify Responsible" for the label as well (discussed with Luc).

Maybe even "sms_notify_responsible" at this parameter is only applied for sms alarms?

"force_xxx" is usually used as a method to force something that is not the default behavior.
It's a bit strange for fields.